README.md

```diff
@@ -31,7 +31,7 @@ https://docs.rs/image
 | WebP   | Lossy(Luma channel only) | No |
 | PNM    | PBM, PGM, PPM, standard PAM | Yes |
 | DDS    | DXT1, DXT3, DXT5 | No |
-| TGA    | Yes | No |
+| TGA    | Yes | Yes |
```

Comment:
I had a question on this part. Since I don't support all of the input ColorType enums, should I explicitly say which ones are supported (ie. ```RGB(8), RGBA(8), Gray(8), GrayA(8)```)

Reply:
That wouldn't hurt, please do.